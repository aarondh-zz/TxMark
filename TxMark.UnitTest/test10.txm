<div>
  There should be "three" printed here: 
  (choose: 2 * 5 - 7)
  (when: 1)[one]
  (when: 2)[two]
  (when: 3)[three]
  (when: 4)[four]
  (when: 5)[five]
  (otherwise:)[I don't know]
</div>
