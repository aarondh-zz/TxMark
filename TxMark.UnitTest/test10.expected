<div>
There should be &quot;three&quot; printed here:
 three
</div>
